pykickstart/commands/bootloader.py

```diff
@@ -514,3 +514,28 @@ def _getParser(self):
 
 class RHEL9_Bootloader(F34_Bootloader):
     pass
+
+class F38_Bootloader(F34_Bootloader):
+    removedKeywords = F34_Bootloader.removedKeywords
+    removedAttrs = F34_Bootloader.removedAttrs
+
+    def __init__(self, writePriority=10, *args, **kwargs):
+        F34_Bootloader.__init__(self, writePriority, *args, **kwargs)
+        self.sdboot = kwargs.get("sdboot", False)
+
+    def _getArgsAsStr(self):
+        ret = F21_Bootloader._getArgsAsStr(self)
```

Comment:
In `F21_Bootloader._getArgsAsStr` it returns only `--disabled` if it is disabled so I think we'll need to do that in all subsequent `_getArgsAsStr` functions as well. That behavior should also be tested in the new F38 test.

Reply:
Ah, good catch. That is a bug for sure!

To avoid this problem in the future, and a lot of DRY, i stretched my python abilities and tweaked future class/method overriding.  :)

But the python docs say that @final is only in python 3.8, and I'm not sure if that is an appropriate assumption here?

Reply:
Yah, so I'm blowing up the unit test with the final... Sorta expected that to happen..